superset/assets/src/visualizations/TreeMap/adaptor.js

```diff
@@ -0,0 +1,55 @@
+import treemap from './treemap';
+
+function buildHiarachey(data, metric, groupby, level) {
```

Comment:
typo `Hierarchy`

Reply:
Potentially can use `stratify` from `d3-hierarchy` instead of implementing by ourselves.

https://github.com/d3/d3-hierarchy#stratify

Reply:
haha! I really need typo check configured in my Visual Studio Code